blog-homepage/src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import MissedArticle from './components/MissedArticle';
+import YourArticle from './components/YourArticle';
+import missedArticles from './_data/missed-articles.json'
+import yourArticles from './_data/your-articles.json';
+import './App.scss';
+
+
+function App() {
+  return (
+    <div className="app">
+      <section>
+        <h1>For You</h1>
+        <hr />
+        <div className="forYou">
+          {yourArticles.map((article, i) => <YourArticle key={i} data={article} />)}
+        </div>
+      </section>
+
+      <section>
+        <h1>In case you missed it</h1>
```

Comment:
A minor nit: typically SEO people will complain that a page should only have _one_ `h1` tag.